.github/workflows/tox-testing.yml

```diff
@@ -0,0 +1,29 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Tox
+      run: pip install tox
+    - name: Run pep8 tests
+      run: tox -vv -e pep8
+    - name: Run unit tests
+      run: tox -vv -e py
+    - name: Run coverage tests
+      run: tox -vv -e cover
+    - name: Run docs tests
+      run: tox -vv -e docs
```

Comment:
Ideally we should run docs and pep8 separately from unit tests.

Reply:
true, I'm going to do a separate job for each